common/api-review/messaging-exp.api.md

```diff
@@ -0,0 +1,32 @@
+## API Report File for "@firebase/messaging-exp"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { FirebaseApp } from '@firebase/app-types-exp';
+import { FirebaseMessaging } from '@firebase/messaging-types-exp';
+import { MessagePayload } from '@firebase/messaging-types-exp';
+import { NextFn } from '@firebase/util';
+import { Observer } from '@firebase/util';
+import { Unsubscribe } from '@firebase/util';
+
+// @public (undocumented)

```

Comment:
Maybe you are planning to do this in another PR but to get documentation, the JSdoc comments need to be added in `src/api.ts`